File: frontend/src/Interfaces/ScheduleTypeModel.ts
 /*export enum ScheduleTypes {
  Morning = "Morning",
  Afternoon = "Afternoon",
  Hybrid = "Hybrid",
}
  */
export enum ScheduleTypes {
  Morning = 1,
  Afternoon = 2,
  Hybrid = 3,
}

export interface ScheduleTypeModel {
  id: string | null;
  scheduleTypes: ScheduleTypes | null;  // Përdor enum-in ScheduleTypes për scheduleTypes
  userId: string;
}

File: Auto-Schedule/Domain/Model/ScheduleTypeModel.cs
using Domain.Entities;
using Domain.Enum;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Model
{
    public class ScheduleTypeModel
    {
        public Guid? Id { get; set; }
        public ScheduleTypes ScheduleTypes { get; set; }
        public Guid UserId { get; set; }
       
    }
}
